apps/i18n/common/en_us.json

```diff
@@ -201,6 +201,7 @@
   "aiChatRateLimitError": "You've sent too many messages in the last minute. Please wait a moment before sending another message.",
   "aiChatTooPersonalUserMessage": "This message has been flagged as too personal.",
   "aiChatUserInputTooLargeMessage": "AI Chat can only process so much information and your last message used too many words. Please clear the chat, or shorten your last message, system prompt, or retrieval(s) to continue chatting.",
+  "aiChatTimeout": "Your request took too long to process. You can try again, or clear the chat to restart the conversation. Using fewer image and PDF attachments may help reduce the time it takes to process your request.",
```

Comment:
minor: would it make sense to have separate error message for multimodal vs non-multimodal requests? we could use the same timeout error code, but pick the error message based on whether the user message had `assets`?

Reply:
Ooh good idea! I'll ask Bryan in the thread where I was asking about message copy.

Reply:
For posterity, ended up being a bit complicated to do this, so we shipped with a single message. Still might be something we want to do in the future -- the simplest way Sanchit and I could come up with would be to do different error codes for timeouts for requests with assets vs. not.

Slack discussion here: https://codedotorg.slack.com/archives/C06FELVTV0Q/p1747093463562829